test/benchmark/package.json

```diff
@@ -2,8 +2,6 @@
   "name": "@vitest/benchmark",
   "private": true,
   "scripts": {
-    "test": "node test.mjs",
-    "testu": "vitest -u",
-    "coverage": "vitest run --coverage"
+    "bench": "vitest bench"
```

Comment:
Benchmarks never fail. I specifically wrote `test.mjs` to correctly test benchmarks output, and now two other files are not tested.

Reply:
Try to run a failing script first. Only after you ensured that tests __can actually__ fail CI, run an actual test.

Right now you are making _assumptions_ that something works, but reality disagrees.

Reply:
Ok, Let me analyze test.js.